langserver/langserver_test.go

```diff
@@ -192,6 +200,10 @@ package main; import "test/pkg"; func B() { p.A(); B() }`,
 				},
 				"is:exported": []string{},
 			},
+			wantWorkspaceReferences: []string{
+				"/src/test/pkg/a.go:0:17 -> /goroot/src/fmt <none>",
```

Comment:
Elsewhere when we specify positions, they are 1-based not 0-based. This is to be consistent with what you see in your editor, error message output from `go`, etc. The fact that this is 0-based is inconsistent/confusing and probably indicates we have 2 different code paths for parsing positions (should just reuse the same one).